app/controllers/identities/password_resets_controller.rb

```diff
@@ -19,12 +20,18 @@ def update
       @identity = Identity.find_by_password_reset_token!(params[:id])
       if @identity.password_reset_sent_at < 2.hours.ago
         redirect_to new_identities_password_reset_path, :alert => "Password reset has expired."
-      elsif @identity.update_attributes(params[:identity])
+      elsif @identity.update_attributes(identity_params)
```

Comment:
`update_attributes` is a hold-over from Rails 3. It still works, but using `update` is the preferred method.